sdk/ml/azure-ai-ml/tests/conftest.py

```diff
@@ -562,15 +564,29 @@ def bodiless_matching(test_proxy):
     set_bodiless_matcher()
 
 
+@pytest.fixture(autouse=True)
+def skip_sleep_in_lro_polling():
+    """Skip sleep in LRO polling for playback mode.
+    In playback mode, we don't need to sleep in LRO polling as no actual remote operations are being performed.
+    """
+    if not is_live():
+        RequestsTransport.sleep = lambda *_, **__: None
+
+
 def pytest_configure(config):
     # register customized pytest markers
     for marker, description in [
         ("e2etest", "marks tests as end to end tests, which involve requests to the server"),
         ("unittest", "marks tests as unit tests, which do not involve requests to the server"),
         ("pipeline_test", "marks tests as pipeline tests, which will create pipeline jobs during testing"),
         ("automl_test", "marks tests as automl tests, which will create automl jobs during testing"),
+        ("core_sdk_test", "marks tests as core sdk tests"),
```

Comment:
is this change comes from another PR?

Reply:
No, it's in this PR, just register some pytest mark to reduce warnings. Can be removed if you have any concern on this